Makefile

```diff
@@ -0,0 +1,20 @@
+
+.PHONY: help deps lint test
+
+help:
+	@echo "Targets:"
+	@echo " - deps: Install required tool and dependencies for building"
+	@echo " - lint: Run golint"
+	@echo " - test: Run unittests"
+
+deps:
+	go get -u github.com/golang/dep/cmd/dep
```

Comment:
For this and the one below, I don't like that it messes with user's binaries. What about one of either:
 * moving dep + golint fetch+setup to a dedicated target
 * make `go get` conditional only if the tool is not already available locally